src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -98,6 +104,124 @@ public static function getExtras()
         return $extras;
     }
 
+    public static function getSystemStatus(): array
+    {
+        $systemStatus = [];
+        $systemStatus['order'] = self::getOrderSystemStatus();
+        $systemStatus['task_processor'] = self::getTaskProcessorStatus();
+        $systemStatus['failed_compatibility_checks'] = self::getFailedCompatibilityChecks();
+
+        return $systemStatus;
+    }
+
+    public static function getFailedCompatibilityChecks(): array
+    {
+        return array_merge(
+            PluginConflictChecker::getPluginsWithConflict(),
+            ServerStatusChecker::getServerIssues(),
+        );
+    }
+
+    public static function getTaskProcessorStatus(): array
+    {
+        $taskProcessorStatus = [];
+
+        $now = current_time('mysql', 1);
+        $calculator = new TaskRateCalculator($now);
+        $processedRate = $calculator->calculateProcessed();
+        $cronRunner = CronOptions::get(CronOptions::RUNNER, CronRegistrar::RUNNER_WPCRON);
+        $postExecutionStatus = CronOptions::get(CronOptions::LAST_EXECUTION_STATUS, CronRegistrar::STATUS_UNEXECUTED);
+        $postExecutionError = CronOptions::get(CronOptions::LAST_POST_EXECUTION_ERROR);
+        $preExecutionDateTime = CronOptions::get(CronOptions::LAST_PRE_EXECUTION_DATE);
+        $invalidRunners = CronOptions::getInvalidRunners();
+
+        $postExecutionSuccessful = CronRegistrar::STATUS_SUCCESS === $postExecutionStatus;
+
+        $lastProcessTaskDate = TaskModel::getLastProcessTaskDate();
+        $lastProcessTaskDateFormatted = null;
+        if ($lastProcessTaskDate) {
+            $lastProcessTaskDateFormatted = \DateTime::createFromFormat('Y-m-d H:i:s', $lastProcessTaskDate)->format(DATE_RFC2822);
+        }
+
+        $taskProcessorStatus['in_queue_quantity'] = TaskModel::count(['status = :status'], ['status' => TaskHandler::STATUS_NEW]);
+        $taskProcessorStatus['processing_p_h'] = $processedRate;
+        $taskProcessorStatus['runner'] = $cronRunner;
+        $taskProcessorStatus['last_execute_date'] = $preExecutionDateTime;
+        $taskProcessorStatus['last_success_date'] = self::getLastSuccessSyncRunDate();
+        $taskProcessorStatus['last_end_date'] = self::getLastSyncRunDate();
+        $taskProcessorStatus['last_task_date'] = $lastProcessTaskDateFormatted;
```

Comment:
are all those dates in same format?

Reply:
Yes, all of them are in `DATE_RFC2822` (D, d M Y H:i:s O) format.